src/components/EventEdit.tsx

```diff
@@ -373,16 +400,17 @@ class EventEdit extends React.PureComponent<PropsType> {
           </div>
         </form>
 
+
```

Comment:
Whitespace changes.

Reply:
should i make a single commit with all the linter white space changes? 
if not ill just remove the changes

Reply:
I don't think this one was added by the linter, did it? Though anyway, even if it was, don't add unrelated whitespace changes to commits.